src/styles/main.scss

```diff
@@ -10,5 +10,7 @@ body {
   box-sizing: border-box;
 }
 
+@import "./utils/variables";
```

Comment:
If you will add bottom dash to all folders, it would be in same code style
```suggestion
@import "./_utils/variables";
```

Reply:
good point